On The Map/Model/Udacity API Requests/Requests/AuthenticationRequest.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  AuthenticationRequest.swift
+//  On The Map
+//
+//  Created by Michal Hus on 2/20/20.
+//  Copyright © 2020 Michal Hus. All rights reserved.
+//
+
+import Foundation
+
+struct AuthenticationRequest {
+    let udacity: Credentials
+}
+
+extension AuthenticationRequest: Codable {}
+
+struct Credentials {
+    let username: String
+    let password: String
+}
+
+extension Credentials: Codable {}
```

Comment:
pop quiz: do you remember the reason why we separated the `Codable` conformance into an extension? 😉

Reply:
It was because of auto-generated init() by xcode for the structs